src/opm/parser/eclipse/EclipseState/Schedule/ArrayDimChecker.cpp

```diff
@@ -43,13 +46,20 @@ namespace {
 
             if (nWells > std::size_t(wdims.maxWellsInField()))
             {
-                std::ostringstream os;
-                os << "Run uses " << nWells << " wells, but allocates at "
-                   << "most " << wdims.maxWellsInField() << " in RUNSPEC "
-                   << "section.  Increase item 1 of WELLDIMS accordingly.";
-
-                ctxt.handleError(Opm::ParseContext::RUNSPEC_NUMWELLS_TOO_LARGE,
-                                 os.str(), guard);
+                const auto& location = wdims.location();
+                if (location) {
+                    std::string fmt_message = fmt::format("Problem with keyword {{keyword}}\n"
+                                                          "In {{file}} line {{line}}\n"
+                                                          "The case has {0} wells - but only {1} are specified in WELLDIMS.\n"
+                                                          "Please increase item 1 in WELLDIMS to at least {1}", nWells, wdims.maxWellsInField());
```

Comment:
Here, I have no idea what {0} and {1} is, but I can guess from the context.

Reply:
OK - it is standard fmt syntax where `{0}` is replaced with the first argument, `{1}` with the second argument and so on; i.e. for this particular example `fmt` will replace
```
{0} -> nWells
{1} -> wdims.maxWellsInField()
```
I can name them.